lib/csv_helper/loader.rb

```diff
@@ -73,5 +54,22 @@ def merge_options(options)
       options.reverse_merge(key_mapping: key_mapping, value_converters: value_converters)
              .reverse_merge(SMARTER_CSV_OPTIONS)
     end
+
+    # default validations are run during import to prevent bad data from going into table,
+    # this runs validations with a context of :load_csv
+    # validations with this context are ran now due either the validation needing data in the table being loaded into
+    # or the validation causes issues on save
```

Comment:
This comment is a little confusing.  Can we clean it up. Suggestion:
```
 # Default validations are run during import, which prevent bad data from being persisted to the database.
 # This method manually runs validations that were declared with a specific validation context (:load_csv).
 # The result is warnings are generated for the end user while the data is allowed to persist to the database. 
```

Reply:
changed